web/src/main/java/org/springframework/security/web/server/authentication/logout/LogoutWebFilter.java

```diff
@@ -85,11 +85,24 @@ public void setLogoutSuccessHandler(
 		this.logoutSuccessHandler = logoutSuccessHandler;
 	}
 
+	/**
+	 * Sets the {@link ServerLogoutHandler}. The default is {@link SecurityContextServerLogoutHandler}.
+	 * @param logoutHandler The handler to use
+	 */
 	public void setLogoutHandler(ServerLogoutHandler logoutHandler) {
 		Assert.notNull(logoutHandler, "logoutHandler must not be null");
 		this.logoutHandler = logoutHandler;
 	}
 
+	/**
```

Comment:
We should not add a getter as it breaks encapsulation

Reply:
The initial review back in May said that we wanted to keep the idea of `DelegatingServerLogoutHandler` to the configuration DSL. In order to do that the configuration DSL needs to be able to reach into the current state of things (i.e. the `LogoutWebFilter`'s current `ServerLogoutHandler` as well as the currently configured `CsrfWebFilter`'s `CsrfTokenRepository`. I'm unsure of how to accomplish that without exposing a getter for that.

Reply:
See https://github.com/spring-projects/spring-security/pull/5379#discussion_r219199535